### PR TITLE
Fix sitting with armor stand and animal mount offsets

### LINKED ISSUE #709: Sitting using Armor Stands seems incorrect
**Describe the bug**

CMI gives players the ability to sit on Stabs and Stairs, but Geyser(Standalone) seems to have you too low. I have pictures showing how it looks for Bedrock clients,

**To Reproduce**

1. Sit using a plugin like CMI on Stairs or Slabs

2. See that your inside the ground/floor

**Expected behavior**

Sit/see in/from the same location as Java

**Screenshots / Videos**

image

**Server Version**

Paper version git-Paper-325 (MC: 1.15.2)

**Geyser Version**

Geyser Standalone version 1.0-SNAPSHOT (git-master-b8a3009)

**Minecraft: Bedrock Edition Version**

1.14.60

**Additional Context**

I have had this issues for awhile.

### PR DESCRIPTION
**Pull request by davchoo**
Fixes #709. I've only tested with GSit and Craftbook's chairs.

### CODE REVIEW

**Comment by TheMasterBlaster** [comment]
I use the CMI plugin for sitting and it seems this works for sitting on Slabs/Stairs or when using the sit command, but it does seem using /ride to ride a player has some issues. 
image

**Comment by davchoo** [comment]
I don't have CMI, but I've tested with GSit and the latest commit should fix it.

**Comment by TheMasterBlaster** [comment]
Just tested again, riding players works when using CMI

File: connector/src/main/java/org/geysermc/connector/entity/living/ArmorStandEntity.java [hunk 1]
```diff
@@ -29,10 +29,12 @@
 import com.github.steveice10.mc.protocol.data.game.entity.metadata.MetadataType;
 import com.nukkitx.math.vector.Vector3f;
 import com.nukkitx.protocol.bedrock.data.entity.EntityData;
+import lombok.Getter;
 import org.geysermc.connector.entity.LivingEntity;
 import org.geysermc.connector.entity.type.EntityType;
 import org.geysermc.connector.network.session.GeyserSession;
 
+@Getter
```
Comment (DoctorMacc) [inline connector/src/main/java/org/geysermc/connector/entity/living/ArmorStandEntity.java:37]: For cleanliness could you move this to just `isMarker`?

**Review by DoctorMacc — APPROVED** [review]

**Pull request merged**